bittensor/metagraph.py

```diff
@@ -240,10 +241,9 @@ def add_args(parser: argparse.ArgumentParser):
         bittensor.wallet.Wallet.add_args( parser )
         bittensor.subtensor.Subtensor.add_args( parser )
         try:
-            parser.add_argument('--metagraph.stale_emit_filter', default=10000, type=int, 
-                                help='''The metagraph filters neurons with last emit beyond this many blocks.
-                                        Note, this is used to trim the graph size,
-                                        but may change your incentive mechanism view.''')
+            parser.add_argument('--metagraph.stale_emit_filter', default=-1, type=int, 
+                                help='''Filter neurons who have not emitted in this number of blocks.
+                                        -1 for no filter.''')
         except:
```

Comment:
Can we also fix this except: pass while we're at it?

Reply:
this except pass just ensures that the args are not added twice. but yes in another commit